include/uv.h

```diff
@@ -1284,6 +1284,20 @@ UV_EXTERN uv_pid_t uv_os_getppid(void);
 UV_EXTERN int uv_os_getpriority(uv_pid_t pid, int* priority);
 UV_EXTERN int uv_os_setpriority(uv_pid_t pid, int priority);
 
+#if defined(__PASE__)
+# define UV_SCHED_PRIORITY_MIN 1
+# define UV_SCHED_PRIORITY_MAX 127
+#endif
```

Comment:
These should not be defined here. If they're intended for public consumption (i.e., for libuv users), then they should always be defined. If they're for internal use, then they should be defined in e.g. src/unix/internal.h.

Reply:
updated